### DISCUSSION TITLE
error when picking up level resources

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#15227] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

title
![изображение](https://github.com/user-attachments/assets/ca633d2c-9e10-4fdc-b751-41a75a11ec13)
![изображение](https://github.com/user-attachments/assets/fd04f5b3-fff0-4cfd-b648-d5256138102c)

### Reproduction steps
be host, detach ore/or plant and pick it up

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

Other

### -

unstable 1.7.4.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Error while reading a message from the server. Entity event data exceeds the size of the buffer (current position: 520, length: 536).
Previous entities:
Human
Error while reading a message from the server. Entity event data exceeds the size of the buffer (current position: 600, length: 616).
Previous entities:
Human
```

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
